### PR TITLE
feat(apu): FLAP OPEN indication until fully closed

### PR DESCRIPTION
**Pull request by davidwalschots**
## Summary of Changes
This further improves the FLAP OPEN indication on the APU ECAM page.

It is displayed when:
1. The flap is fully open
2. The flap was fully open and is closing, but not fully closed.
3. The flap was fully open, started closing, but started opening again before fully closing.

Additionally this PR ensures that while the FLAP OPEN indication is shown, the APU N and EGT are displayed (instead of amber XX); even when the APU N = 0 and MASTER SW is OFF.

## Additional context

Discord username (if different from GitHub): SjotgunSjonnie#9623

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -41,20 +41,20 @@
         <Sound WwiseEvent="LEFT_WING_SCRAPE" ContinuousStopDelay="0.25" FadeOutType="2" FadeOutTime="0.25" WwiseData="true" NodeName="AILERON_RIGHT001"/>
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:41]: Should not be here even if valid changes

**Review by Benjozork — APPROVED** [review]
LGTM

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml (hunk 1 above)
Comment (davidwalschots) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:41]: I'm not sure what you mean. `APU_FLAP_OPEN` was renamed to `A32NX_APU_FLAP_OPEN_PERCENTAGE`.

**Comment by St54Kevin** [comment]
Category: QA Tester
Name: St54Kevin#3583
Date of testing: 02/02/2021
Version of the sim: 1.12.13.0
PR Tested: #3280
Tier of testing: n/a
Changes to observe: Flap open indication

Testing technique: 
Steps you've made to test the Pull Request:

1. Started C&D 
2. Started APUw without bleed
3. shut it down
4. observe indications
5. Started with bleed
6. shut down again
7. observed 2min cooldown

Media:
image

Issues (If Any): I observed the above display after shutting down. It looks like the display is kind of broken isnt it?

Overall Rating: LGTM

Conclusions of the testing: Does exactly what this PR is suppose to do. Please have a look at my screenshot

**Comment by davidwalschots** [comment]
The screenshot you posted is correct. I agree with you that the real-life situation looks broken 🤣.

**Pull request merged**